modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -100,19 +99,16 @@ const loadIcons = (scriptLoader: ScriptLoader, editor: Editor, suffix: string) =
   });
 };
 
-const loadPlugins = (editor: Editor, settings: EditorSettings, suffix: string) => {
-  if (Type.isArray(settings.plugins)) {
-    settings.plugins = settings.plugins.join(' ');
-  }
-
-  Tools.each(settings.external_plugins, function (url, name) {
+const loadPlugins = (editor: Editor, suffix: string) => {
+  // These lines should be changed to a getParam and some type of setParam once such an API is available.
+  Tools.each(editor.getParam('external_plugins'), (url: string, name: string): void => {
```

Comment:
This should be in `Settings`.

Reply:
external_plugins and plugins are different enough?